docs/developer/workflows.md

```diff
@@ -28,3 +28,16 @@ These workflows will create a [report of deprecated terms with Mondo xrefs](../r
 2. `mapped-deprecated-terms-artefacts`: Creates a reports/%_mapped_deprecated_terms.robot.template.tsv for all ontologies.
 3. `mapped-deprecated-terms-docs`: Creates a report of statistics for mapped deprecated terms (docs/reports/mapped_deprecated.md) and pages for each ontology which list their deprecated terms with existing xrefs in Mondo.
 4. `mapped-deprecated-terms`: Creates a report of statistics for mapped deprecated terms (docs/reports/mapped_deprecated.md) and pages for each ontology which list their deprecated terms with existing xrefs in Mondo. Also creates a reports/%_mapped_deprecated_terms.robot.template.tsv for all ontologies.
+
+
+## Exclusions
```

Comment:
@matentzn @sabrinatoro FYI: Docs
I added some docs for how to update the OMIM exclusions with non-diseases, but I the docs for running the exclusion tables generally were missing, so I added those.

Reply:
@sabrinatoro I don't know if you should / want to review this. I updated the `config/omim_exclusions.tsv` to add non-diseases. Then I ran the exclusion pipeline to generate new artefacts that were updated:

- `reports/omim_exclusion_reasons.robot.template.tsv`
- `reports/omim_term_exclusions.txt`

Feel free to check any of these files. More importantly, you will probably want to look at this [OMIM PR](https://github.com/monarch-initiative/omim/pull/91) where I tagged you on a comment.

I also see this related issue, which I might as well do this week as well. It would fit in quite easily with what I've done here:
- https://github.com/monarch-initiative/mondo-ingest/issues/169

Reply:
@sabrinatoro @matentzn I also see this related issue, which I might as well do this week as well. It would fit in quite easily with what I've done here.

- https://github.com/monarch-initiative/mondo-ingest/issues/169

Reply:
Correct me if I am wrong @joeflack4. You produce a list of non-disease terms at ingest time, which you then _parse_ here, resulting in additional exclusions. 

If this is correct, I think this is slightly hacky. I would prefer:

1. You tag the respective non-diseases with a standard annotation triple in the ingest (e.g. `?x MONDO:exclusionReason MONDO:nonDisease`). 
2. Your generic exclusion script will look for this tag, and use it to produce additional intensional exclusions.
3. No custom scripts for ingest necessary. We can use the same system for all future ingests, tagging classes at ingest time with this triple, and it will just work.

Reply:
This makes sense. I thought about this to see if there were any disadvantages, and the only one I could think of is that, given the way we set this up now, this makes it less clear to the curator that these things are being excluded, since we have intensional exclusion files, e.g. `config/omim_exclusions.tsv` that are being maintained by the curator / Sabrina at the moment, and these are used to create extensional exclusion files, e.g. `reports/omim_term_exclusions.txt` and `reports/omim_exclusion_reasons.robot.template.tsv`.

This is still my top priority issue which I will likely pick up on again tomorrow, and I can move forward with this change.

Reply:
donezo